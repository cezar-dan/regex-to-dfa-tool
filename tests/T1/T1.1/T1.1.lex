 
SPACE
0
0,' ',1
1,' ',2
1

0
ZEROS
0
0,'0',1
1,'0',1
1