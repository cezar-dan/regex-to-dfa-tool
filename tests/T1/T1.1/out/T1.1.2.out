SPACE  
ZEROS 0000
SPACE  